botx/bot/bot.py

```diff
@@ -299,6 +303,30 @@ async def remove_users_from_chat(
         payload = BotXAPIRemoveUserRequestPayload.from_domain(chat_id, huids)
         await method.execute(payload)
 
+    async def add_admin_roles(
```

Comment:
Can you explain `add_admin_roles` method name plz? Maybe `add_admin` or `add_chat_admin`?